libs/blocks/quiz/quiz.js

```diff
@@ -279,43 +276,48 @@ const App = ({
     return optionItem && optionItem[prop] ? optionItem[prop] : '';
   };
 
-  return html`<div class="quiz-container">
-                  <${StepIndicator} 
-                    currentStep=${currentStep} 
-                    totalSteps=${totalSteps} 
-                    prevStepIndicator=${prevStepIndicator}
-                    top="${true}" />
+  if (isDataLoaded) {
```

Comment:
If you do any early return, you don't need to put this into an else/if
```suggestion
  if(!isDataLoaded) return ''
```

Reply:
Done. Thanks!